docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -0,0 +1,202 @@
+
+MapLoader/MapStore is configured in the `Map` resource. Before configuring the MapLoader/MapStore for `Map` resource be sure to add the MapLoader/MapStore classes into Hazelcast members' CLASSPATH using User Code Deployment feature.
+
+Below is the configuration options for MapStore in the `Map` resource. You can find more detailed information in the xref:api-ref.adoc#mapstoreconfig[API Reference] page.
+
+|===
+|Field|Description
+
+|initialMode
+|Sets the initial entry loading mode.
+
+|className
+|Name of your class implementing MapLoader and/or MapStore interface.
+
+|writeDelaySeconds
+|Number of seconds to delay the storing of entries.
+
+|writeBatchSize
+|Used to create batches when writing to map store.
+
+|writeCoealescing
+|It is meaningful if you are using write behind in MapStore. When it is set to true, only the latest store operation on a key during the write-delay-seconds will be reflected to MapStore.
+
+|propertiesSecretName
+|Properties can be used for giving information to the MapStore implementation. Properties will be taken from the secret with the given name.
+
+|===
+
+=== Example MapStore Configuration
+
+Below is an example `Map` resource with MapStore configuration. 
+
+.Example configuration
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/map-map-store.yaml[]
+----
+
+== Configuring Executor Services with the Operator
+
+Hazelcast offers `IExecutorService` which is an implementation for `java.util.concurrent.ExecutorService`. With `IExecutorService`, you can execute tasks asynchronously and perform other useful tasks. To use Executor Service, users need to implement `Callable` or `Runnable` interfaces and deploy the deployed classes into Hazelcast cluster using User Code Deployment feature.
+
+Executor Service has two more variants Durable Executor Service and Scheduled Executor Service. More information can be found in the link:https://docs.hazelcast.com/hazelcast/latest/computing/executor-service[documentation].
+
+Executor service configuration can be added into `Hazelcast` with the following configuration. 
+
+[tabs]
+====
+Executor Service::
++
+--
+More information can be found in the xref:api-ref.adoc#executorserviceconfiguration[API Reference] page.
+
+|===
+|Field|Description
+
+|name
+|The name of the executor service
+
+|poolSize
+|The number of executor threads per member.
+
+|queueCapacity
+|Task queue capacity of the executor.
+
+|===
+--
+
+Durable Executor Service::
++
+--
+More information can be found in the xref:api-ref.adoc#durableexecutorserviceconfiguration[API Reference] page.
+
+|===
+|Field|Description
+
+|name
+|The name of the executor service
+
+|poolSize
+|The number of executor threads per member.
+
+|durability
+|Durability of the executor.
```

Comment:
Could a more detailed description be given? What effect do the numerical values for this option have?